src/Help/TicketSystem/TicketMessageRepository.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EtoA\Help\TicketSystem;
+
+use Doctrine\DBAL\Connection;
+use EtoA\Admin\AdminUserRepository;
+use EtoA\Core\AbstractRepository;
+use EtoA\User\UserRepository;
+
+class TicketMessageRepository extends AbstractRepository
+{
+    private AdminUserRepository $adminUserRepo;
+    private UserRepository $userRepo;
+
+    public function __construct(
+        Connection $connection,
+        AdminUserRepository $adminUserRepo,
+        UserRepository $userRepo
+    ) {
+        parent::__construct($connection);
+        $this->adminUserRepo = $adminUserRepo;
+        $this->userRepo = $userRepo;

```

Comment:
Different repositories should not depend on each other

Reply:
What is a better way to solve it in this case?

Reply:
As far as I can see the two repositories are only used to fetch the user/admin name. So can move the getAuthorNick method to a service

Reply:
Done in b38ab27 and 3f4eff8